src/node/coinstats.cpp

```diff
@@ -75,9 +77,33 @@ bool GetUTXOStats(CCoinsView* view, CCoinsStats& stats, const std::function<void
         pcursor->Next();
     }
     if (!outputs.empty()) {
-        ApplyStats(stats, ss, prevkey, outputs);
+        ApplyStats(stats, hash_obj, prevkey, outputs);
     }
-    stats.hashSerialized = ss.GetHash();
+
+    FinalizeHash(hash_obj, stats);
+
     stats.nDiskSize = view->EstimateSize();
     return true;
 }
+
+bool GetUTXOStats(CCoinsView *view, CCoinsStats& stats, CoinStatsHashType hash_type, const std::function<void()>& interruption_point)
+{
+    switch(hash_type) {
+        case(CoinStatsHashType::HASH_SERIALIZED): {
```

Comment:
ed4c7302298385b797000740960b06ff91082ce8

nit, this format is not used. Also add space after `switch` above.

Reply:
;)
https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy

Reply:
done